cmd/compare.go

```diff
@@ -130,6 +130,9 @@ func compare(provider string, oldCommit string, newCommit string) error {
 			violations = append(violations, vs...)
 		}
 
+		if f.Outputs == nil {
```

Comment:
Should you also check if `newFunc.Outputs == nil` in what follows? (This is done in the analogous inputs code, just before).